src/routes/login/LoginView.vue

```diff
@@ -272,7 +272,12 @@ export default {
       this.username = '';
       this.pw = '';
       this.loading = false;
-      this.$router.push('/');
+      if (this.authStore.loginRedirect) {
+        this.$router.push(this.authStore.loginRedirect);
+        this.authStore.loginRedirect = null;
+      } else {
+        this.$router.push('/');
+      }
```

Comment:
Since both use router push, you can likely consolidate this approach. What's the purpose of nulling out `this.authStore.loginRedirect`? it feels like code smell to me.

```
const redirectPath = this.authStore.loginRedirect ?? '/';
this.authStore.loginRedirect = null; // figure out some way to not need to do this
this.$router.push(redirectPath);
```